selendroid-standalone/src/main/java/io/selendroid/standalone/server/grid/SelfRegisteringRemote.java

```diff
@@ -136,7 +146,7 @@ private JSONObject getConfiguration() throws JSONException {
       configuration.put("proxy", "org.openqa.grid.selenium.proxy.DefaultRemoteProxy");
     }
     configuration.put("role", "node");
-    configuration.put("registerCycle", 5000);
+    configuration.put("registerCycle", config.getRegisterCycle());
```

Comment:
I believe @frolix's case is more unusual...

@frolix wrote:
> I'd like to suggest to set the default value to 0 as it will be inline with @asm89 point above about disabling the recurrence behavior and with original grid's sources SelfRegisteringRemote.java, line 191.

this keeps behavior compatibility. :+1: